CHANGELOG.rst

```diff
@@ -42,6 +44,9 @@ Completions
 
 Improved terminal support
 ^^^^^^^^^^^^^^^^^^^^^^^^^
+- ``man terminal-compatibility`` documents required and optional features.
+- fish supports `set NO_COLOR 1 <https://no-color.org/>`_ to completely disable color output (syntax highlighting, prompt, builting output, error messages).
+  However grayscale colors (for autosuggestions) are still enabled.
```

Comment:
A different solution is to turn off autosuggestions and force the pager selection to use underline.

That's probably more in the spirit of NO_COLOR.

Reply:
I have not used the `NO_COLOR` behavior myself much yet,
but I do use the None theme, so it's possible that I want `NO_COLOR`.
I wouldn't use it without autosuggestions.

No strong opinion on the pager selection; I'd think that reverse video is allowed in `NO_COLOR` but haven't checked.

It's probably true that the `NO_COLOR` spec doesn't want anything but black-on-white (or the other way round).
To not openly violate NO_COLOR, maybe we should use a different name? `set -g fish_grayscale 1`?